tasks/sass/index.js

```diff
@@ -17,9 +18,16 @@ module.exports = config => {
     .then(() => new Promise((resolve, reject) => {
       const aliases = {};
       if (config.theme) {
+        console.log('Applying theme: ' + config.theme);
         aliases.$$theme = `hof-theme-${config.theme}`;
       }
-      sass(config.sass.src, { aliases }, (err, result) => err ? reject(err) : resolve(result.css));
+
+      sass.render({
+        file: config.sass.src,
+        importer: importer({ aliases })
```

Comment:
needs to be 
```
sass.render({
        file: config.sass.src,
        importer: importer({ aliases }),
       aliases
      }, function (err, result) {
```

Reply:
Thanks @Alex-Swann . Are you able to shed some light on what this is actually doing? I can't find any documentation related to this.

Reply:
Resolved.